spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/SparkCassandraITFlatSpecBase.scala

```diff
@@ -13,4 +20,41 @@ trait SparkCassandraITWordSpecBase extends WordSpec with SparkCassandraITSpecBas
 trait SparkCassandraITAbstractSpecBase extends AbstractSpec with SparkCassandraITSpecBase
 
 trait SparkCassandraITSpecBase extends Suite with Matchers with SharedEmbeddedCassandra with SparkTemplate with BeforeAndAfterAll {
+
+  def getKsName = {
+    val className = this.getClass.getSimpleName
+    val suffix =  StringUtils.splitByCharacterTypeCamelCase(className.filter(_.isLetterOrDigit)).mkString("_")
+    s"test_$suffix".toLowerCase()
+  }
+
+  val ks = getKsName
+
+  implicit val ec = SparkCassandraITSpecBase.ec
+
+  def doAsync(units: Future[Unit]*): Unit = {
```

Comment:
A blocking method called "Async". I guess the purpose of this method is to just wait for all the futures to complete, right? So why not name it `awaitAll` ?